lib/gringotts/gateways/trexle.ex

```diff
@@ -340,6 +340,13 @@ defmodule Gringotts.Gateways.Trexle do
     }
   end
 
+  defp respond({:ok, %{status_code: code, body: _body}}) when code in [401] do
```

Comment:
I'm skeptical about this clause... looking at the mock response you added, the response body is empty.
We must still set the `Response.raw` key to `""`.

Reply:
Done